dotnet-engine/Yggdrasil.Engine.Tests/YggdrasilEngineTest.cs

```diff
@@ -6,7 +6,25 @@
 using Newtonsoft.Json.Linq;
 using Yggdrasil;
 using Yggdrasil.Test;
+using System.Text.Json.Serialization;
 
+public class TestVariantReadModel
+{
+    public TestVariantReadModel(string name, Payload? payload, bool enabled, bool featureEnabled)
+    {
+        Name = name;
+        Payload = payload;
+        Enabled = enabled;
+        FeatureEnabled = featureEnabled;
+    }
+
+    public string Name { get; set; }
+    public Payload? Payload { get; set; }
+    public bool Enabled { get; set; }
+
+    [JsonPropertyName("feature_enabled")]
```

Comment:
This one seems to be missing in the Variant class?

Reply:
Yes, by design, the Variant class can and should handle "featureEnabled", this class only exists so we can materialise the "feature_enabled" property in the spec tests